src/components/MultiHopTrade/hooks/useQuoteValidationErrors.ts

```diff
@@ -0,0 +1,78 @@
+import { useHopHelper } from 'components/MultiHopTrade/hooks/useHopHelper'
+import { useIsTradingActive } from 'components/MultiHopTrade/hooks/useIsTradingActive'
+import { useReceiveAddress } from 'components/MultiHopTrade/hooks/useReceiveAddress'
+import { ActiveQuoteStatus } from 'components/MultiHopTrade/types'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { walletSupportsChain } from 'hooks/useWalletSupportsChain/useWalletSupportsChain'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { isTruthy } from 'lib/utils'
+import {
+  selectFirstHopNetworkFeeCryptoPrecision,
+  selectFirstHopSellAsset,
+  selectFirstHopTradeDeductionCryptoPrecision,
+  selectLastHopBuyAsset,
+  selectLastHopNetworkFeeCryptoPrecision,
+  selectMinimumSellAmountCryptoBaseUnit,
+  selectSellAmountCryptoBaseUnit,
+} from 'state/slices/tradeQuoteSlice/selectors'
+import { useAppSelector } from 'state/store'
+
+type QuoteValidationErrors = ActiveQuoteStatus[]
+
+export const useQuoteValidationErrors = (): QuoteValidationErrors => {
```

Comment:
nitpick but im a fan of only creating a custom type when the alternative is more confusing - prefer minimising the number of indirections for simple types like an array

Reply:
I resonate with that reasoning, fixed in https://github.com/shapeshift/web/pull/4889/commits/3c2dfb0db4a92e0f1a79c3401a5342d15ab69a42